client/blocks/app-banner/index.jsx

```diff
@@ -131,9 +132,21 @@ class AppBanner extends Component {
 			}
 		}
 
-		//TODO: update when deferred deep links are available
 		if ( this.isiOS() ) {
-			return 'itms://itunes.apple.com/us/app/wordpress/id335703880?mt=8';
+			switch ( currentSection ) {
```

Comment:
If we want to verify that we return the correct app links, consider adding a unit test. It's fine to pull this switch out into it's own function (makes it easier to test).

```js
//...
if ( this.isiOS() ) {
    return getIOSDeepLinks( currentSection ); 
}
//...
}

function getIOSDeepLinks( currentSection ) {
    switch ( currentSection ) {
      //...
}
```
And in the test file:
```
//...
	test( 'iOS Deep links return correct URIs for READER', () => {
        expect( '/' ).toEqual( https://apps.wordpress.com/get#read );
        expect( '/read/feeds/12345/posts/6789' ).toEqual( 'https://apps.wordpress.com/get#read/feeds/12345/posts/6789' );
	} );
    test( 'iOS Deep links return correct URIs for STATS', () => {
        //...
	} );
//...
```

Reply:
Thanks – I've pulled out a function and added some tests!